CHANGELOG.rst

```diff
@@ -7,7 +7,16 @@ Changelog
 
 Features
 ~~~~~~~~
-- 1928: implement changelog generation using `gitchangelog` [Aleksandr]
+- 1970: Add optional schema endpoint. [Aleksandr]
+- 1620: Introduce time based triggers (#1961) [Daniil]
+- 1918: Implement basic authentication for `client` (#1942) [Daniil]
+- 1726: Implement a release PR workflow. (#1932) [Aleksandr Petrosyan]
+- 1815: Make query responses more type-structured (#1867) [s8sato]
```

Comment:
Why does @s8sato is mentioned by it's Github name and others by their real names? Also sometimes you use full name and someties just first name. I think we need one style. At least for codeowners

Reply:
> Why does @s8sato is mentioned by it's Github name and others by their real names?

Because of how he signs-off his commits. 

```
Signed-off-by: s8sato <49983831+s8sato@users.noreply.github.com>
```

> Also sometimes you use full name and someties just first name

This is because some of us (myself included) have used 
```
git config --global user.name Aleksandr Petrosyan
```
when we were supposed to do 

```
git config --global user.name "Aleksandr Petrosyan"
```

I don't mind it having only my first name, but if you want to always have your full name, you can amend the signoff on your commits.